scrapy/core/engine.py

```diff
@@ -90,56 +90,63 @@ def start(self) -> Deferred:
         yield self._closewait
 
     def stop(self) -> Deferred:
-        """Stop the execution engine gracefully"""
+        """
+        Gracefully stop the execution engine
+        """
```

Comment:
Your docstring style changes seem to go against https://www.python.org/dev/peps/pep-0257/

Reply:
Interesting, I wasn't aware of that particular point. I don't really like it, but I won't go stubborn on this. Thanks!